### ISSUE TITLE
yum.qubes-os.org is down

### ISSUE THREAD

**Issue by edwintorok** [QubesOS/qubes-issues#3811]
### Qubes OS version:

Qubes release 4.0 (R4.0)

### Affected component(s): 

qubes-dom0-update

---

### Steps to reproduce the behavior:

`sudo qubes-dom0-update`

### Expected behavior:

Able to reach repository

### Actual behavior:

Failed to synchronize cache for repo 'qubes-dom0-current', disabling.
Failed to synchronize cache for repo 'qubes-templates-itl', disabling.

### General notes:

```
wget http://yum.qubes-os.org
--2018-04-10 23:47:46--  http://yum.qubes-os.org/
Resolving yum.qubes-os.org (yum.qubes-os.org)... 82.94.215.165
Connecting to yum.qubes-os.org (yum.qubes-os.org)|82.94.215.165|:80... failed: Connection refused.
```

---

**Comment by woju** [comment]
Yes, this was a real failure related to an update, which went bad. Should be working now.
